src/pudl/extract/extractor.py

```diff
@@ -66,6 +61,16 @@ def _load_csv(self, package: str, filename: str) -> pd.DataFrame:
             importlib.resources.files(package) / filename, index_col=0, comment="#"
         )
 
+    def _load_column_maps(self, column_map_pkg: str) -> dict:
+        """Create a dictionary of all column mapping CSVs to use in get_column_map()."""
+        column_dict = {}
+        for res_path in importlib.resources.files(column_map_pkg).iterdir():
+            # res_path is expected to end with ${page}.csv
+            if res_path.suffix == ".csv":
+                column_map = self._load_csv(column_map_pkg, res_path.name)
+                column_dict[res_path.stem] = column_map
+        return column_dict
+
```

Comment:
is this just a formality seeing as we don't have any column maps for this dataset?

Reply:
This is moving the existing load step into a separate method in the base class, so that i'm able to subclass and change it in the VCE extractor.